src/coreclr/jit/emit.h

```diff
@@ -2464,10 +2464,10 @@ class emitter
     unsigned char emitOutputLong(BYTE* dst, size_t val);
     unsigned char emitOutputSizeT(BYTE* dst, size_t val);
 
-    unsigned char emitOutputByte(BYTE* dst, unsigned __int64 val);
-    unsigned char emitOutputWord(BYTE* dst, unsigned __int64 val);
-    unsigned char emitOutputLong(BYTE* dst, unsigned __int64 val);
-    unsigned char emitOutputSizeT(BYTE* dst, unsigned __int64 val);
+    unsigned char emitOutputByte(BYTE* dst, uint64_t val);
```

Comment:
Similar question on whether cases like `unsigned char` have any plans to become `uint8_t` in the future

Reply:
I think bulk find&replace of `{unsigned} char` with `{u}int8_t` would be questionable. `char` is the right type to use when dealing with characters.

Of course, folks are welcomed to do targeted improvements and refactoring. For this specific method, I think `unsigned` or `size_t` as return type would be better than `uint8_t`.

Reply:
> char is the right type to use when dealing with characters.

Right, this was specifically about `unsigned char` and `signed char`. If it's actual characters I would expect its "just `char`" and should stay as is.

More just interested in at as a potential future improvement.